Replace.java

```diff
@@ -0,0 +1,21 @@
+
+package simplenotepad;
+
+import java.awt.event.ActionEvent;
+import javax.swing.JOptionPane;
+import javax.swing.JTextPane;
+
+
+public class Replace extends EventChooser{
+	
+	public Replace(JTextPane pane) {
+		super(pane);
+	}
+	

```

Comment:
this is really great and I like the way you simplify it  
I think I did my replace method differently